### DISCUSSION TITLE
Pocketbase 0.23

### DISCUSSION THREAD

**Discussion by andrewpmoore** [pockethost/pockethost#411] [Q&A]
I'm keen to join the flounders as I'd been playing with hooks a local instance as I'm new to pb.
The hooks I've written are based on the new 0.23 structure. Do you know how soon this might become available as 0.22 seems to be the latest I can see.
Love the idea of it just being managed for me.

**Comment by benallfree** [comment]
We have a [v23](https://github.com/pockethost/pockethost/tree/feat/v23) branch under testing. It could be released as soon as a few days, I’m just trying to figure out how to do a phased release so I can roll back if needed. There are some significant changes to v23 and not sure we can catch them all except to just release it and see what happens :)

**Reply by andrewpmoore** [reply]
Thanks for the reply, that's good to hear. 👍

**Reply by andrewpmoore** [reply]
I've just bought flounders 🥳
